Makefile

```diff
@@ -442,6 +442,22 @@ $(BINDIR)/gen-versions: $(shell find ./hack/gen-versions -type f)
 	sh -c '$(GIT_CONFIG_SSH) \
 	go build -o $(BINDIR)/gen-versions ./hack/gen-versions'
 
+## Generate a ClusterServiceVersion package.
+# E.g. make gen-csv VERSION=1.6.2 PREV_VERSION=0.0.0
+#
+# VERSION: the operator version to generate a CSV for.
+# PREV_VERSION: the operator version that this CSV will replace. If there is
+#               no previous version, use 0.0.0
+.PHONY: gen-csv
+gen-csv: hack/bin/operator-sdk-$(OPERATOR_SDK_VERSION)
+	hack/gen-csv/csv.sh
```

Comment:
I think we should containerize this. Would `$(CONTAINERIAED) hack/gen-csv/csv.sh` work?
Same for gen-bundle below?

Reply:
Actually, using the gobuild container won't work because it doesn't have docker itself. The csv.sh script uses `docker pull` and `docker inspect`. Maybe it's still okay since we're using the operator-sdk binary in hack/ ?

Reply:
I want to use the containerized because I'm concerned about using the operator-sdk specifically. That's because I think it also depends on the go version (at least from previous experience).

Reply:
Yeah you're right. I've addressed that with b5caf0b

Once I get https://github.com/projectcalico/go-build/pull/115 approved+merged+released, that should fix the build